src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
+CREATE TABLE `manufacturers` (
```

Comment:
It would be better if you use 'CREATE DATABASE IF NOT EXISTS; instead of only 'CREATE DATABASE'

Reply:
It would be better if you use 'CREATE TABLE IF NOT EXISTS ; instead of only 'CREATE TABLE'